src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/LabelComponentSpec.scala

```diff
@@ -16,7 +16,8 @@ import scala.util.Random
 class LabelComponentSpec extends TestComponent with FlatSpecLike with CommonTestData with GcsPathUtils {
 
   "LabelComponent" should "save, get,and delete" in isolatedDbTest {
-    val c1 = Cluster(
+    val cluster1 =
```

Comment:
Typically, code blocks line the one below is right-indented since they're the continuation of the line above. Same for the other instances.

Reply:
Gotcha. I think this is a weird IntelliJ thing that happened because I didn't get to update this file yet